tests/test_utils.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for utilities."""
+
+import numpy as np
+import pytest
+from pytest import param
+
+from baybe.utils.numerical import closest_element
+
+
+@pytest.mark.parametrize(
+    "as_ndarray", [param(False, id="list"), param(True, id="array")]
+)
+@pytest.mark.parametrize(
+    ("array", "target", "expected"),
+    [
+        param(0, 0.1, 0, id="0D"),
```

Comment:
(minor)
since you always try to find 0 this seems a bit over-engineered and one could simply get rid of `expected` and perhaps even `target`
so id either i) get rid of it or ii) dont get rid of it but make test actually such that they use different target-expected pairs

Reply:
Agree, that happens when you make huge plans but then change direction along the way :D 
Removed the two arguments and also changed the target to a nonzero value, because zero is always some sort of edge case ...